# Termodinámica Química. Ciclo 01-2019.
# Resolución de examen final.

## Problema 1.
El ciclohexano puede producirse por la hidrogenación del benceno. Suponga que el proceso se lleva a cabo en dos reactores conectados en serie. El primer reactor opera a 340 ºC y 5 bar, mientras que el segundo lo hace a 265 ºC y 5 bar. La alimentación es una mezcla con proporción de 10 moles de hidrógeno por mol de benceno. La reacción sucede completamente en fase gaseosa.

a) ¿Cuál es la composición de equilibrio de la salida del segundo reactor?

b) ¿Porqué considera que se usan dos reactores en lugar de uno solo?

c) Si los reactores  operaran a 1 bar, ¿cuánto más o cuánto menos producto se obtendría?

d) ¿Recomendaría usted diluir la alimentación con un inerte para aumentar la obtención de ciclohexano?

e) Sus compañeros Jairo y Francia habían trabajado ya con esta reacción mientras estudiaban, por lo que han escrito la reacción como en sus apuntes. Jairo había escrito:

$$C_6 H_6 + 3H_2 \rightarrow C_6H_{12}$$

Mientras que Francia había escrito:

$$2C_6H_6 + 6 H_2 \rightarrow 2C_6H_{12}$$

### Solución.

(NOTA: las soluciones a continuación son extensas ya que tratan de mostrar explícitamente todos los cálculos realizados. Muchos de los cálculos a continuación podían hacerse automáticamente con la hoja de Excel proporcionada en clase).

### Literal a)

Información:

&nbsp;|$C_6H_6$|$H_2$|$C_6H_{12}$
---:|---:|---:|---:
$\nu_i$|-1|-3|1
$\Delta H^0_f$|82930|0|-123140
$\Delta G^0_f$|129665|0|31920
A|-0.206|3.249|-3.846
B|39.064x 10<sup>-3</sup>|0.422x10<sup>-3</sup>|63.249x10<sup>-3</sup>
C|-13.301x10<sup>-6</sup>|0|-20.928<sup>-6</sup>
D|0|0.083x10<sup>5</sup>|0

Determinación de la constante de equilibrio para el primer reactor:

In [1]:
global K P0 P sumavi
datos=[-1 -3 1;
82930 0 -123140;
129665 0 31920;
-0.206 3.249 -3.876;
39.064e-3 0.422e-3 63.249e-3;
-13.301e-6 0 -20.928e-6;
0 0.083e5 0];
R=8.314
T0=298.15
T=340+273.15
tao=T/T0
P0=1
P=5
sumavi=sum(datos(1,:))
deltaH0=sum(datos(1,:).*datos(2,:))
deltaG0=sum(datos(1,:).*datos(3,:))
deltaA=sum(datos(1,:).*datos(4,:))
deltaB=sum(datos(1,:).*datos(5,:))
deltaC=sum(datos(1,:).*datos(6,:))
deltaD=sum(datos(1,:).*datos(7,:))
K0=exp(-deltaG0/(R*T0))
K1=exp(deltaH0/(R*T0)*(1-T0/T))
K2=exp(deltaA*(log(tao)-(tao-1)/tao)+0.5*deltaB*T0*(tao-1)^2/tao+(1/6)*deltaC*T0^2*((tao-1)^2*(tao+2))/tao+0.5*deltaD/T0^2*(tao-1)^2/tao^2)
K=K0*K1*K2



R =  8.3140
T0 =  298.15
T =  613.15
tao =  2.0565
P0 =  1
P =  5
sumavi = -3
deltaH0 = -206070
deltaG0 = -97745
deltaA = -13.417
deltaB =  0.022919
deltaC =   -7.6270e-06
deltaD = -24900
K0 =    1.3339e+17
K1 =    2.8311e-19
K2 =  0.29752
K =  0.011236


Para determinar las composiciones en el equilibrio, hay que determinar el valor de la coordenada de la reacción. Para ello, pueden escribirse las composiciones en el equilibrio como funciones de $\xi$:

$$y_{i}=\frac{n_{i,0}+\nu_i \xi}{n_{t,0}+\xi \Sigma \nu_i}$$

Si se usa una alimentación de 1 mol de benceno y 10 moles de hidrógeno:

$$\Sigma \nu_i=-3 \\
n_{t,0}=11 \\
y_{benceno}=\frac{1-\xi}{11-3\xi} \\
y_{hidrógeno}=\frac{10-3\xi}{11-3\xi} \\
y_{ciclohexano}=\frac{\xi}{11-3\xi}$$

Ecuación a resolver:

$$K\left(\frac{P^0}{P} \right)^{\Sigma \nu_i}=\prod y_i^{\nu_i}$$

In [2]:
moles_iniciales1=[1 10 0]
function coord=coord(x)
    global K P0 P sumavi
    coord(1)=x(1)-(1-x(4))/(11-3*x(4));
    coord(2)=x(2)-(10-3*x(4))/(11-3*x(4));
    coord(3)=x(3)-x(4)/(11-3*x(4));
    coord(4)=K*(P0/P)^(sumavi)-x(1)^(-1)*x(2)^(-3)*x(3);
endfunction
guess=[0.0012 0.8755 0.1233 0.99];
coordenada_equilibrio1=fsolve("coord",guess)(4)
composiciones_equilibrio1=fsolve("coord",guess)(1:3)
moles_totales_equilibrio1=11-3*coordenada_equilibrio1
moles_i_equilibrio1=moles_totales_equilibrio1*composiciones_equilibrio1
conversiones1=(moles_iniciales1-moles_i_equilibrio1)./moles_iniciales1*100


moles_iniciales1 =

    1   10    0

coordenada_equilibrio1 =  0.50146
composiciones_equilibrio1 =

   0.052502   0.894688   0.052810

moles_totales_equilibrio1 =  9.4956
moles_i_equilibrio1 =

   0.49854   8.49561   0.50146

conversiones1 =

   50.146   15.044     -Inf



Constante de equilibrio para el segundo reactor:

In [3]:
global K P0 P sumavi
datos=[-1 -3 1;
82930 0 -123140;
129665 0 31920;
-0.206 3.249 -3.876;
39.064e-3 0.422e-3 63.249e-3;
-13.301e-6 0 -20.928e-6;
0 0.083e5 0];
R=8.314
T0=298.15
T=265+273.15
tao=T/T0
P0=1
P=5
sumavi=sum(datos(1,:))
deltaH0=sum(datos(1,:).*datos(2,:))
deltaG0=sum(datos(1,:).*datos(3,:))
deltaA=sum(datos(1,:).*datos(4,:))
deltaB=sum(datos(1,:).*datos(5,:))
deltaC=sum(datos(1,:).*datos(6,:))
deltaD=sum(datos(1,:).*datos(7,:))
K0=exp(-deltaG0/(R*T0))
K1=exp(deltaH0/(R*T0)*(1-T0/T))
K2=exp(deltaA*(log(tao)-(tao-1)/tao)+0.5*deltaB*T0*(tao-1)^2/tao+(1/6)*deltaC*T0^2*((tao-1)^2*(tao+2))/tao+0.5*deltaD/T0^2*(tao-1)^2/tao^2)
K=K0*K1*K2

R =  8.3140
T0 =  298.15
T =  538.15
tao =  1.8050
P0 =  1
P =  5
sumavi = -3
deltaH0 = -206070
deltaG0 = -97745
deltaA = -13.417
deltaB =  0.022919
deltaC =   -7.6270e-06
deltaD = -24900
K0 =    1.3339e+17
K1 =    7.9187e-17
K2 =  0.40847
K =  4.3147


In [4]:
Composiciones en el equilibrio para el segundo reactor:

moles_iniciales2=moles_i_equilibrio1
function coord=coord(x)
    global K P0 P sumavi
    coord(1)=x(1)-(0.47276-x(4))/(9.4183-3*x(4));
    coord(2)=x(2)-(8.41828-3*x(4))/(9.4183-3*x(4));
    coord(3)=x(3)-(0.47415+x(4))/(9.4183-3*x(4));
    coord(4)=K*(P0/P)^(sumavi)-x(1)^(-1)*x(2)^(-3)*x(3);
endfunction
guess=[1.9999e-5 0.8750 0.1183 0.4726];
coordenada_equilibrio2=fsolve("coord",guess)(4)
composiciones_equilibrio2=fsolve("coord",guess)(1:3)
moles_totales_equilibrio2=9.4183-3*coordenada_equilibrio2
moles_i_equilibrio2=moles_totales_equilibrio2*composiciones_equilibrio2
conversiones2=(moles_iniciales2-moles_i_equilibrio2)./moles_iniciales2*100

error: 'Composiciones' undefined near line 1 column 1
moles_iniciales2 =

   0.49854   8.49561   0.50146

coordenada_equilibrio2 =  0.47015
composiciones_equilibrio2 =

   3.2621e-04   8.7512e-01   1.1792e-01

moles_totales_equilibrio2 =  8.0079
moles_i_equilibrio2 =

   0.0026123   7.0078368   0.9442977

conversiones2 =

   99.476   17.512  -88.309



Las composiciones en el equilibrio son por tanto:

$$y_{benceno}=3.2621 \times 10^{-4}\\
y_{hidrógeno}=8.7512 \times 10^{-1} \\
y_{ciclohexano}=1.1792 \times 10^{-1}$$

### Literal b)

Como puede observarse, la conversión del benceno alcanzada en el primer reactor es ligeramente superior al 50%, por lo que aún queda casi la mitad del benceno sin reaccionar. El segundo reactor se utiliza para convertir el benceno restante, pero no puede operar en las mismas condiciones, puesto que entonces se estaría alimentando una mezcla que ya está en equilibrio, por lo que el segundo reactor opera en condiciones diferentes que permitan desplazar la reacción hacia la derecha. La conversión alcanzada en el segundo reactor es mayor al 99%, por lo que la unión de los dos reactores permite un consumo casi completo del benceno.

### Literal c)

Para el primer reactor:

In [49]:
global K P0 P sumavi
datos=[-1 -3 1;
82930 0 -123140;
129665 0 31920;
-0.206 3.249 -3.876;
39.064e-3 0.422e-3 63.249e-3;
-13.301e-6 0 -20.928e-6;
0 0.083e5 0];
R=8.314
T0=298.15
T=340+273.15
tao=T/T0
P0=1
P=1
sumavi=sum(datos(1,:))
deltaH0=sum(datos(1,:).*datos(2,:))
deltaG0=sum(datos(1,:).*datos(3,:))
deltaA=sum(datos(1,:).*datos(4,:))
deltaB=sum(datos(1,:).*datos(5,:))
deltaC=sum(datos(1,:).*datos(6,:))
deltaD=sum(datos(1,:).*datos(7,:))
K0=exp(-deltaG0/(R*T0))
K1=exp(deltaH0/(R*T0)*(1-T0/T))
K2=exp(deltaA*(log(tao)-(tao-1)/tao)+0.5*deltaB*T0*(tao-1)^2/tao+(1/6)*deltaC*T0^2*((tao-1)^2*(tao+2))/tao+0.5*deltaD/T0^2*(tao-1)^2/tao^2)
K=K0*K1*K2

moles_iniciales1=[1 10 0]
function coord=coord(x)
    global K P0 P sumavi
    coord(1)=x(1)-(1-x(4))/(11-3*x(4));
    coord(2)=x(2)-(10-3*x(4))/(11-3*x(4));
    coord(3)=x(3)-x(4)/(11-3*x(4));
    coord(4)=K*(P0/P)^(sumavi)-x(1)^(-1)*x(2)^(-3)*x(3);
endfunction
guess=[0.0012 0.8755 0.1233 0.99];
coordenada_equilibrio1=fsolve("coord",guess)(4)
composiciones_equilibrio1=fsolve("coord",guess)(1:3)
moles_totales_equilibrio1=11-3*coordenada_equilibrio1
moles_i_equilibrio1=moles_totales_equilibrio1*composiciones_equilibrio1
conversiones1=(moles_iniciales1-moles_i_equilibrio1)./moles_iniciales1*100

R =  8.3140
T0 =  298.15
T =  613.15
tao =  2.0565
P0 =  1
P =  1
sumavi = -3
deltaH0 = -206070
deltaG0 = -97745
deltaA = -13.417
deltaB =  0.022919
deltaC =   -7.6270e-06
deltaD = -24900
K0 =    1.3339e+17
K1 =    2.8311e-19
K2 =  0.29752
K =  0.011236
moles_iniciales1 =

    1   10    0

coordenada_equilibrio1 =  0.0083657
composiciones_equilibrio1 =

   9.0355e-02   9.0888e-01   7.6225e-04

moles_totales_equilibrio1 =  10.975
moles_i_equilibrio1 =

   0.9916343   9.9749028   0.0083656

conversiones1 =

   0.83657   0.25097      -Inf



In [54]:
global K P0 P sumavi
datos=[-1 -3 1;
82930 0 -123140;
129665 0 31920;
-0.206 3.249 -3.876;
39.064e-3 0.422e-3 63.249e-3;
-13.301e-6 0 -20.928e-6;
0 0.083e5 0];
R=8.314
T0=298.15
T=265+273.15
tao=T/T0
P0=1
P=1
sumavi=sum(datos(1,:))
deltaH0=sum(datos(1,:).*datos(2,:))
deltaG0=sum(datos(1,:).*datos(3,:))
deltaA=sum(datos(1,:).*datos(4,:))
deltaB=sum(datos(1,:).*datos(5,:))
deltaC=sum(datos(1,:).*datos(6,:))
deltaD=sum(datos(1,:).*datos(7,:))
K0=exp(-deltaG0/(R*T0))
K1=exp(deltaH0/(R*T0)*(1-T0/T))
K2=exp(deltaA*(log(tao)-(tao-1)/tao)+0.5*deltaB*T0*(tao-1)^2/tao+(1/6)*deltaC*T0^2*((tao-1)^2*(tao+2))/tao+0.5*deltaD/T0^2*(tao-1)^2/tao^2)
K=K0*K1*K2

moles_iniciales2=moles_i_equilibrio1
function coord=coord(x)
    global K P0 P sumavi
    coord(1)=x(1)-(0.9916343-x(4))/(10.975-3*x(4));
    coord(2)=x(2)-(9.9749028-3*x(4))/(10.975-3*x(4));
    coord(3)=x(3)-(0.0083656+x(4))/(10.975-3*x(4));
    coord(4)=K*(P0/P)^(sumavi)-x(1)^(-1)*x(2)^(-3)*x(3);
endfunction
guess=[0.0111 0.8791 0.1098 0.9];
coordenada_equilibrio2=fsolve("coord",guess)(4)
composiciones_equilibrio2=fsolve("coord",guess)(1:3)
moles_totales_equilibrio2=10.975-3*coordenada_equilibrio2
moles_i_equilibrio2=moles_totales_equilibrio2*composiciones_equilibrio2
conversiones2=(moles_iniciales2-moles_i_equilibrio2)./moles_iniciales2*100

R =  8.3140
T0 =  298.15
T =  538.15
tao =  1.8050
P0 =  1
P =  1
sumavi = -3
deltaH0 = -206070
deltaG0 = -97745
deltaA = -13.417
deltaB =  0.022919
deltaC =   -7.6270e-06
deltaD = -24900
K0 =    1.3339e+17
K1 =    7.9187e-17
K2 =  0.40847
K =  4.3147
moles_iniciales2 =

   0.9916343   9.9749028   0.0083656

coordenada_equilibrio2 =  0.74151
composiciones_equilibrio2 =

   0.028584   0.885709   0.085695

moles_totales_equilibrio2 =  8.7505
moles_i_equilibrio2 =

   0.25013   7.75038   0.74987

conversiones2 =

     74.776     22.301  -8863.727



La conversión ha caído por debajo del 1% en el primer reactor, por lo que prácticamente todo el benceno pasa al segundo. Ahí, la conversión es de aproximadamente 75%. Por lo tanto, dado que a 5 bar se consume prácticamente todo el benceno, operando a 1 bar se obtiene 25% menos producto.

### Literal d) 

Para el primer reactor:

In [56]:
global K P0 P sumavi
datos=[-1 -3 1 0;
82930 0 -123140 0;
129665 0 31920 0;
-0.206 3.249 -3.876 3.28;
39.064e-3 0.422e-3 63.249e-3 0.593e-3;
-13.301e-6 0 -20.928e-6 0;
0 0.083e5 0 0.04e5];
R=8.314
T0=298.15
T=340+273.15
tao=T/T0
P0=1
P=5
sumavi=sum(datos(1,:))
deltaH0=sum(datos(1,:).*datos(2,:))
deltaG0=sum(datos(1,:).*datos(3,:))
deltaA=sum(datos(1,:).*datos(4,:))
deltaB=sum(datos(1,:).*datos(5,:))
deltaC=sum(datos(1,:).*datos(6,:))
deltaD=sum(datos(1,:).*datos(7,:))
K0=exp(-deltaG0/(R*T0))
K1=exp(deltaH0/(R*T0)*(1-T0/T))
K2=exp(deltaA*(log(tao)-(tao-1)/tao)+0.5*deltaB*T0*(tao-1)^2/tao+(1/6)*deltaC*T0^2*((tao-1)^2*(tao+2))/tao+0.5*deltaD/T0^2*(tao-1)^2/tao^2)
K=K0*K1*K2

moles_iniciales1=[1 10 0 1]
function coord=coord(x)
    global K P0 P sumavi
    coord(1)=x(1)-(1-x(5))/(12-3*x(5));
    coord(2)=x(2)-(10-3*x(5))/(12-3*x(5));
    coord(3)=x(3)-x(5)/(12-3*x(5));
    coord(4)=x(4)-1/(12-3*x(5));
    coord(5)=K*(P0/P)^(sumavi)-x(1)^(-1)*x(2)^(-3)*x(3);
    
endfunction
guess=[0.0108 0.7850 0.0968 0.1075 0.9];
coordenada_equilibrio1=fsolve("coord",guess)(5)
composiciones_equilibrio1=fsolve("coord",guess)(1:4)
moles_totales_equilibrio1=12-3*coordenada_equilibrio1
moles_i_equilibrio1=moles_totales_equilibrio1*composiciones_equilibrio1
conversiones1=(moles_iniciales1-moles_i_equilibrio1)./moles_iniciales1*100

R =  8.3140
T0 =  298.15
T =  613.15
tao =  2.0565
P0 =  1
P =  5
sumavi = -3
deltaH0 = -206070
deltaG0 = -97745
deltaA = -13.417
deltaB =  0.022919
deltaC =   -7.6270e-06
deltaD = -24900
K0 =    1.3339e+17
K1 =    2.8311e-19
K2 =  0.29752
K =  0.011236
moles_iniciales1 =

    1   10    0    1

coordenada_equilibrio1 =  0.43033
composiciones_equilibrio1 =

   0.053195   0.813241   0.040184   0.093379

moles_totales_equilibrio1 =  10.709
moles_i_equilibrio1 =

   0.56967   8.70900   0.43033   1.00000

conversiones1 =

   4.3033e+01   1.2910e+01         -Inf   2.7219e-06



Para el segundo reactor:

In [60]:
global K P0 P sumavi
datos=[-1 -3 1 0;
82930 0 -123140 0;
129665 0 31920 0;
-0.206 3.249 -3.876 3.28;
39.064e-3 0.422e-3 63.249e-3 0.593e-3;
-13.301e-6 0 -20.928e-6 0;
0 0.083e5 0 0.04e5];
R=8.314
T0=298.15
T=265+273.15
tao=T/T0
P0=1
P=5
sumavi=sum(datos(1,:))
deltaH0=sum(datos(1,:).*datos(2,:))
deltaG0=sum(datos(1,:).*datos(3,:))
deltaA=sum(datos(1,:).*datos(4,:))
deltaB=sum(datos(1,:).*datos(5,:))
deltaC=sum(datos(1,:).*datos(6,:))
deltaD=sum(datos(1,:).*datos(7,:))
K0=exp(-deltaG0/(R*T0))
K1=exp(deltaH0/(R*T0)*(1-T0/T))
K2=exp(deltaA*(log(tao)-(tao-1)/tao)+0.5*deltaB*T0*(tao-1)^2/tao+(1/6)*deltaC*T0^2*((tao-1)^2*(tao+2))/tao+0.5*deltaD/T0^2*(tao-1)^2/tao^2)
K=K0*K1*K2

moles_iniciales2=moles_i_equilibrio1
function coord=coord(x)
    global K P0 P sumavi
    coord(1)=x(1)-(0.56967-x(5))/(10.709-3*x(5));
    coord(2)=x(2)-(8.709-3*x(5))/(10.709-3*x(5));
    coord(3)=x(3)-(0.43033+x(5))/(10.709-3*x(5));
    coord(4)=x(4)-1/(10.709-3*x(5));
    coord(5)=K*(P0/P)^(sumavi)-(x(1)^(-1))*(x(2)^(-3))*x(3);
endfunction
guess=[0.0011 0.7785 0.1097 0.1108   0.56];
coordenada_equilibrio2=fsolve("coord",guess)(5)
composiciones_equilibrio2=fsolve("coord",guess)(1:4)
moles_totales_equilibrio2=moles_totales_equilibrio1-3*coordenada_equilibrio2
moles_i_equilibrio2=moles_totales_equilibrio2*composiciones_equilibrio2
conversiones2=(moles_iniciales2-moles_i_equilibrio2)./moles_iniciales2*100

R =  8.3140
T0 =  298.15
T =  538.15
tao =  1.8050
P0 =  1
P =  5
sumavi = -3
deltaH0 = -206070
deltaG0 = -97745
deltaA = -13.417
deltaB =  0.022919
deltaC =   -7.6270e-06
deltaD = -24900
K0 =    1.3339e+17
K1 =    7.9187e-17
K2 =  0.40847
K =  4.3147
moles_iniciales2 =

   0.56967   8.70900   0.43033   1.00000

coordenada_equilibrio2 =  0.56529
composiciones_equilibrio2 =

   4.3476e-04   7.7810e-01   1.1046e-01   1.1095e-01

moles_totales_equilibrio2 =  9.0131
moles_i_equilibrio2 =

   0.0039185   7.0131396   0.9956332   1.0000322

conversiones2 =

   9.9312e+01   1.9473e+01  -1.3136e+02  -3.2180e-03



Dado que para esta reacción $\Sigma \nu_i$ es negativo, la adición del inerte disminuye la conversión, como puede observarse en la conversión en el primer reactor en comparación con el caso anterior sin inerte presente.
Al igual que en el caso original, el segundo reactor consume el benceno restante. Esto, por cierto, confirma la imporancia del segundo reactor, que incluso en estas condiciones permite el consumo casi completo del benceno. Aún así, la conversión se reduce al incluir el inerte, por lo que no es recomendable hacer la dilución.

### Literal e)

Para la constante de equilibrio, puede verse por medio de la siguiente ecuación:

$$K \left( \frac{P^0}{P} \right)^{\Sigma \nu_i}=\prod y_i^{\nu_i} \\
K=\left( \frac{P^0}{P} \right)^{-\Sigma \nu_i} \prod y_i^{\nu_i}$$

Para Jairo sería:

$$ K=\left( \frac{P^0}{P} \right)^{-(-3)} y_1^{-1} y_2^{-3} y_3$$

Para Francia sería:

$$ K=\left( \frac{P^0}{P} \right)^{-(-6)} y_1^{-2} y_2^{-6} y_3^2$$

Es decir, la constante de equilibrio calculada por Francia sería el cuadrado de la calculada por Jairo.

Para la coordenada de reacción:

$$\frac{dn_i}{\nu_i}=d\xi$$

Por lo que para cualquier $dn_i$, la coordenada de reacción obtenida por Francia será la mitad de la calculada por Jairo.

Finalmente, dado que los coeficientes de Francia son el doble de los de Jairo, $\Delta G^0$ y $\Delta H^0$ serán el doble de los calculados por Jairo.

Ambos contestarán correctamente al literal a) del problema, ya que a pesar de las diferencias entre $\xi$, *K*, $\Delta H^0$ y $\Delta G^0$, las composiciones de equilibrio serán las mismas para ambos, ya que para una reacción química no importan los coeficientes utilizados siempre y cuando cumpla con la conservación de la materia.

## Problema 2.

Se construye una celda con un electrodo de plomo y un electrodo de hidrógeno. La celda opera a 298.15 K y 1 bar. La concentración de iones Pb2+ en su semicelda es de 1 molar.

a) Escriba las semireacciones y la reacción global de la celda. 

b) Calcule el potencial estándar de la celda.

c) Calcule el pH en la semicelda del hidrógeno cuando el potencial es de 0.244 V.

NOTA: para especies en solución, en estas condiciones, la relación $\frac{\hat{f_i}}{f_i^0}$ puede aproximarse a su molaridad, mientras que para especies sólidas es igual a 1.

### Literal a)

Las reacciones escritas en una tabla de potenciales estándar para los electrodos en cuestión son:

$$Pb^{2+}(ac)+2e^- \rightarrow Pb(s)  \hspace{1cm} -0.13 V \\
2 H^+(ac)+2 e^- \rightarrow H_2 (g) \hspace{1cm} 0.0 V$$

El potencial de reducción más grande es el del hidrógeno, por lo tanto él será el que se reduzca (cátodo). Por lo tanto la semireacción de reducción es:

$$2 H^+(ac)+2 e^- \rightarrow H_2 (g)$$

En tanto que la de oxidación será:

$$Pb(s) \rightarrow Pb^{2+}(ac)+2e^-$$

Invertida ya que será una oxidación.

La reacción global será la combinación de ambas:

$$2 H^+(ac)+2 e^- \rightarrow H_2 (g) \\
Pb(s) \rightarrow Pb^{2+}(ac)+2e^- \\
\text{____________________________} \\
2H^+(ac) + Pb(s) \rightarrow Pb^{2+}(ac)+H_2 (g)$$

### Literal b)

El potencial estándar de la celda es:

$$E_{celda}^0=E_{cátodo}^0-E_{ánodo}^0 \\
E_{celda}^0=0.0 V - (-0.13 V) \\
E_{celda}^0=0.13 V $$

### Literal c)

El potencial de una celda se puede determinar por medio de la ecuación de Nernst:

$$E_{celda}=E_{celda}^0-\frac{RT}{zF} \text{ln} \prod \left( \frac{\hat{f_i}}{f_i^0} \right)^{\nu_i}$$

Donde la relación $\frac{\hat{f_i}}{f_i^0}$ se aproxima a la molaridad para especies en solución (como el ión plomo), y es igual a la unidad para especies sólidas (como el plomo sólido). Por otra parte, al operar la celda a 298.15 y 1 bar, es posible suponer gases ideales, por lo que dicha relación será igual a la presión para el caso de especies gaseosas (como el hidrógeno), quedando como única incógnita la molaridad del ión hidrógeno:

$$E_{celda}=E_{celda}^0-\frac{RT}{zF} \text{ln} \left( \frac{([Pb^{2+}](P)}{[H⁺]^2(1)} \right)$$

Sustituyendo los valores conocidos:

$$E_{celda}=E_{celda}^0-\frac{RT}{zF} \text{ln} \left( \frac{([1](1)}{[H⁺]^2(1)} \right) $$

Despejando:

$$[H⁺]=\sqrt{\frac{1}{exp(-\frac{zF}{RT} (E_{celda}-E_{celda}^0))}}$$

Siendo *z* los electrones transferidos en la reacción: 2; F, la constante de Faraday: 96485.34 C/mol y T, la temperatura de la celda: 298.15 K. Sustituyendo:

In [5]:
z=2
F=96500%96485.34
T=298.15
E=0.244
E0=0.13
R=8.314
molaridad_h=sqrt(1/(exp(-z*F/(R*T)*(E-E0))))
pH=-log10(molaridad_h)

z =  2
F =  96500
T =  298.15
E =  0.24400
E0 =  0.13000
R =  8.3140
molaridad_h =  84.606
pH = -1.9274


El pH negativo indica una altísima concentración de iones hidrógeno en la solución de esa semicelda.

## Problema 3.

El etanol (1) y el hexano (2) forman dos capas líquidas a 75 ºC, que a su vez están en contacto con una fase vapor. Las composiciones de las fases líquidas son:

$$x_1^\alpha=0.9098 \\
x_1^\beta=0.0902$$

Las presiones de saturación a 75 ºC son:

$$P^{sat}_1=0.888\text{ bar} \\
P^{sat}_2=1.223 \text{ bar} $$

Determine el (o los) parámetro(s) para un modelo apropiado para GE y encuentre:

a) La presión del sistema.

b) La composición de la fase vapor.

### Literal a)

El modelo más simple para representar $G^E$ es el de Margules de 1 parámetro:

$$\frac{G^E}{x_1x_2RT}=A$$

Para el cual:

$$\gamma_1=exp(Ax_2^2) \\
\gamma_2=exp(Ax_1^2)$$

Tal parámetro puede determinarse al resolver la ecuación de equilibrio de alguno de los componentes. Por ejemplo, para el componente 1:

$$x_1^\alpha \gamma_1^\alpha=x_1^\beta \gamma_1^\beta \\
x_1^\alpha exp(A {x_2^\alpha}^2)=x_1^\beta exp(A {x_2^\alpha}^2) \\
0.9098 exp(A (1-0.9098)^2)=0.0902 exp(A (1-0.0902)^2)$$




In [89]:
function marg1=marg1(x)
    marg1=0.9098*exp(x*(1-0.9098)^2)-0.0902*exp(x*(1-0.0902)^2);
endfunction
A=fsolve("marg1",1)

A =  2.8199


Conocido el valor de A es posible calcular la presión. Si se considera la fase $\alpha$ junto con la fase vapor:

$$y_1 P=x_1^\alpha \gamma_1^\alpha P^{sat}_1 \\
y_2 P =x_2^\alpha \gamma_2^\alpha P^{sat}_2 $$

Por lo tanto:

$$P=x_1^\alpha \gamma_1^\alpha P^{sat}_1+x_2^\alpha \gamma_2^\alpha P^{sat}_2 \\
P=0.9098 exp(2.8199(1-0.9098)^2) (0.888)+(1-0.9098) exp(2.8199(0.9098^2)) (1.223)$$



In [90]:
P=0.9098*exp(2.8199*(1-0.9098)^2)*0.888+(1-0.9098)*exp(2.8199*0.9098^2)*1.223

P =  1.9652


Con la presión y las ecuaciones de equilibrio recién escrita, se determina la composición del vapor:

$$y_1=\frac{x_1^\alpha \gamma_1^\alpha P^{sat}_1}{P} \\
y_1=\frac{0.9098 (exp(2.8199(1-0.9098)^2) (0.888)}{1.9652}$$

In [92]:
y1=(0.9098*exp(2.8199*(1-0.9098)^2)*0.888)/1.9652
y2=1-y1

y1 =  0.42065
y2 =  0.57935


## Problema 4.

Se diluyen 500 lbm de una solución de ácido sulfúrico al 40% en peso a 140 ºF con 200 lbm de agua pura a 100 ºF. 

a) ¿Cuál es la concentración de la solución resultante?

b) ¿Cuál es el efecto térmico de esa dilución si la temperatura de la solución resultante es 100 ºF?

c) Si la dilución se lleva a cabo adiabáticamente, ¿cuál será la temperatura de la solución resultante?

### Literal a)

Un balance de masa.

![](esquema.png)

Para el ácido sulfúrico:

$$500(0.4)=Ax$$

Balance global:

$$500+200=A$$

In [95]:
function bal=bal(x)
    bal(1)=500*0.4-x(1)*x(2);
    bal(2)=500+200-x(1);
endfunction
fsolve("bal",[100 0.5])

ans =

   700.00000     0.28571



### Literal b)

Un balance de energía:

$$\Delta K + \Delta P +\Delta H=Q+W \\
\Delta H =Q $$
$$700H_{28.57\%,100 ºF}-500H_{40\%,140 ºF}-200H_{agua,100 ºF}=Q$$

Los valores de entalpía pueden leerse del diagrama entalpía vs composición:

![](hvsx.png)

Completando la ecuación:

$$700 (-40)-500(-46)-200 (64)=Q \\
Q=-17800\text{ BTU}$$





### Literal c)

Como el mezclado es adiabático, puede resolverse el problema gráficamente. Si se unen los puntos representando las soluciones que se están mezclando, la solución resultante estará en la línea recta que conecta esos puntos. Como ya se conoce la composición de la solución resultante, puede trazarse una vertical hasta cortar esa recta y leer el valor de la temperatura.

![](mixad.png)

La temperatura es aproximadamente 130 ºF.

## BONO.

Describa esta imagen con una ecuación:

![](bonoeq.png)

Solución:

$$\Sigma \nu_i \mu_i=0$$

(Equilibrio químico)